app/models/edition/translatable.rb

```diff
@@ -64,6 +68,12 @@ def change_translations_locale_if_primary_locale_changed
     end
   end
 
+  def remove_other_translations_if_primary_locale_no_longer_english
```

Comment:
Does this mean it's not possible for a document to have a "primary" locale of Welsh and a secondary locale of English? Is that a scenario that could exist? (I forget exactly but I suspect not because Whitehall assumes default language is English?)

Reply:
I see now that the scenario I described shouldn't happen. I think we could add a `primary_locale_changed?` conditional on this to make it slightly safer though. I'll have a go at committing a change that does that.

Reply:
For some reason I couldn't get `primary_locale_changed?` to work. But I don't think my concerns are valid because:

```
irb(main):007:0> Edition.publicly_visible.where.not(primary_locale: :en).select { |e| e.translations.count > 1 }.count
=> 0
```